backend/src/api/AuthController.ts

```diff
@@ -28,15 +36,87 @@ export default class AuthController {
             legacyHeaders: false
         });
 
+        // 5 requests per hour
+        const resetPaswsordLimiter = rateLimit({
```

Comment:
I think we need to limit by key (in this case — the email parameter). The rationale for that: currently a malicious party can DDOS this endpoint with reset requests containing arbitrary emails, and that will prevent legitimate users from resetting passwords.

Another, better solution would be to add limiter only for the failed requests (see sign-in for example), and fail request (in controller/manager) in the following cases:
* when requested email is not found in DB
* when new request comes too soon (we store the date of the last generated request)
* when too many active codes are stored in DB